torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -623,6 +623,45 @@ static void eraseListConstruct(Block* block, int opset_version) {
   eraseListConstruct(block->return_node(), opset_version);
 }
 
+static void eraseListUnpack(Block* block, int opset_version);
+
+static void eraseListUnpack(Node* n, int opset_version) {
+  for (auto b : n->blocks()) {
```

Comment:
if `n->kind() == prim::ListUnpack`, n should have no blocks, right?
Can we write the logic as:
```cpp
if (n->kind() == prim::ListUnpack) {
  ...
  return;  // early return
}
for (auto b : n->blocks()) {
  ...
}
```

Reply:
what's the benefit of it... this adds assumption that prim::ListUnpack should not include any blocks.